CHANGELOG.md

```diff
@@ -38,7 +38,7 @@ _None._
 
 ### New Features
 
-_None._
+- Make `publish_private_pod` support `--allow-warnings`. [#86]
```

Comment:
💭 Worth introducing Danger to this repo to remind us not to forget to add those entries next time? 🤔